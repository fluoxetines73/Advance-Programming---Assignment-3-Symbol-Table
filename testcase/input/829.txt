INSERT f_FF string
ASSIGN f_FF 'abcABC'
INSERT x string
ASSIGN x 'HelloWorld'
BEGIN
INSERT e_123_2r number
BEGIN
BEGIN
BEGIN
LOOKUP e_123_2r
BEGIN
BEGIN
END
BEGIN
PRINT
END
END
END
END
END
END