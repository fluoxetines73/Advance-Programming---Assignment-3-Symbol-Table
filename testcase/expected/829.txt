success
success
success
success
success
1
f_FF//0 x//0 e_123_2r//1